adminPanel/addformDB.php

```diff
@@ -0,0 +1,27 @@
+<?php
+$db = new PDO("mysql:host=192.168.20.20;dbname=PortfolioProjects", 'root', '');
+$db -> setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+if(!empty($_POST['title'])) {
+    $title = filter_var($_POST['title'], FILTER_SANITIZE_STRING);
+    if(!empty($_POST['link'])) {
+        $site_url = filter_var($_POST['link'], FILTER_VALIDATE_URL);
+    }
+$title = $_POST['title'];
+$link = $_POST['link'];
+$image = 'images/' . $_POST['image'];
```

Comment:
no if statement to check you have an image?

Reply:
done